### ISSUE TITLE
https://share.cryptozombies.io/

### ISSUE THREAD

**Issue by danielmeyers34** [loomnetwork/cryptozombies-lesson-code#26]
This link is broken.. Just a heads up, this link seems to go to a code page - no info shown. 
https://share.cryptozombies.io/

**Comment by th3pac1fist** [comment]
@danielmeyers34 yeah I noticed that to. I just finished the first lesson and wanted to share my zombie I created out, but it doesn't populate the page with the image :(.
